assets/src/wizards/subscriptions/index.js

```diff
@@ -77,6 +78,8 @@ class SubscriptionsWizard extends Component {
 						disabled
 					/>
 				</Card>
+
+				<NewspackButton isPrimary value="Continue" />
```

Comment:
Let's move the button into the `Card` so we see it side-by-side with other example components.